code/modules/uplink/uplink_items.dm

```diff
@@ -2310,6 +2310,17 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	restricted_roles = list("Research Director", "Scientist", "Roboticist")
 	include_objectives = list(/datum/objective/hijack, /datum/objective/martyr) //yogs // >Increase price to reduce grief > limit it to hijack only :think:
 
+/datum/uplink_item/role_restricted/nanomachinesson
+	name = "Nanomachine Heart"
+	desc = "We saw that our rival nanotrasen had been making advances in nanites technlology so we one oneupped	them in the labs \
+			Our souped up heart does double the nanite regen of nanotrasens heart all while also being emp proof, and ontop of that\
```

Comment:
Needs a space at the end or it'll merge with the next line with no space in between